doc/source/groups_of_atoms.rst

```diff
@@ -64,6 +64,104 @@ Similarly, an :class:`~MDAnalysis.core.groups.Atom` has direct knowledge of the
+    ATOM   1036  HB1 ALA B 122     -28.658  20.583 -13.783  1.00  0.00           H
+    ATOM   1037  HB2 ALA B 122     -27.998  18.983 -14.196  1.00  0.00           H
+    ATOM   1038  HB3 ALA B 122     -27.788  19.635 -12.554  1.00  0.00           H
+    ATOM   1039  N   GLY B 123     -25.713  21.969 -12.223  1.00 41.18           N
+    ATOM   1040  CA  GLY B 123     -25.550  23.204 -11.460  1.00 41.40           C
+    ATOM   1041  C   GLY B 123     -24.309  24.018 -11.745  1.00 45.74           C
+    ATOM   1042  O   GLY B 123     -24.349  25.234 -11.601  1.00 46.81           O
+    ATOM   1043  H   GLY B 123     -25.290  21.133 -11.845  1.00  0.00           H
+    ATOM   1044  HA3 GLY B 123     -25.593  22.976 -10.395  1.00  0.00           H
+    ATOM   1045  HA2 GLY B 123     -26.430  23.831 -11.600  1.00  0.00           H
+    TER
+    """)
+
+Use case: Sequence of residues by segment
+-----------------------------------------
+In order to select only ATOM record types and get a list of residues by segment, one needs to call:
+
+.. ipython:: python
+
+    u = mda.Universe(pdb, format="PDB")
+    residues = list()
+    for seg in u.segments:
+        p_seg = seg.atoms.select_atoms("record_type ATOM")
+        residues.append(p_seg.residues.resnames)
+    residues
+
+As expected, HETATM record type lines are not considered.
+
+Although syntactically correct, accessing residues via segments (to avoid looping through segments),
+would lead to **unexpected** output
+
+.. ipython:: python
+
+    selected_atoms = u.select_atoms("record_type ATOM")
+    all_residues = selected_atoms.segments.resnames
+    all_residues
+
+containing residues both for ATOM and HETATM record types; this construct means
+"give me all residue names from segments in which there is at least one of the selected atoms".
+
+Use case: Atoms list grouped by residues
+----------------------------------------
+In order to list all the heavy protein backbone and sidechain atoms in every residue, one needs to call:
+
+.. ipython:: python
+
+    u = mda.Universe(pdb, format="PDB")
+    atoms_in_residues = list()
+    for seg in u.segments:
+        p_seg = seg.atoms.select_atoms("record_type ATOM and not name H*")
+        for p_res in p_seg.residues:
+            atoms_in_residues.append(p_seg.select_atoms(f"resid {p_res.resid} and resname {p_res.resname}").names)
+    atoms_in_residues
+
+As expected, HETATM record type lines and hydrogen atoms are not considered.
+
+Although syntactically correct, accessing atoms via residues (to avoid looping through residues),
+would lead to **unexpected** output
+
+.. ipython:: python
+
+    all_atoms_in_residues = list()
+    for seg in u.segments:
+        p_seg = seg.atoms.select_atoms("record_type ATOM and not name H*")
+        all_atoms_in_residues.append(p_seg.residues.names)
+    all_atoms_in_residues
+
+containing hydrogen atoms; this construct means "give me all atom names from residues
```

Comment:
Explanation is good.

"please" is not needed here, IMO — it doesn't fit the overall writing style of the UG

Reply:
Done